### ISSUE TITLE
show data structures and algorithms!

### ISSUE THREAD

**Issue by jhoffman222** [jhoffman222/DataStructures#2]

**Comment by Keerthivardhan1** [comment]
can you please elaborate the issue

**Comment by jhoffman222** [comment]
please show any data structure or algorithm in any language

**Comment by Keerthivardhan1** [comment]
ok
shell I add stack ,queue ,and minimum spanning tree algorithm

**Comment by jhoffman222** [comment]
that would be great!

**Comment by Keerthivardhan1** [comment]
can you please assign this issue to me

**Comment by jhoffman222** [comment]
done

**Comment by Keerthivardhan1** [comment]
shell i add this algorithm in txt file are .java extension file 
or
shell i add both algorithm and  data structure in .java extension file

**Comment by jhoffman222** [comment]
please add them with .java extension thanks

**Comment by Keerthivardhan1** [comment]
will this prs be considered in hacktoberfest

**Comment by jhoffman222** [comment]
yes all pr's are labeled as hacktoberfest-accepted as long as they are not spam and show some kind of algorithm or data structure

**Comment by Keerthivardhan1** [comment]
shell I add tree data structure in java lang

**Comment by jhoffman222** [comment]
yes you may

**Comment by Keerthivardhan1** [comment]
ok

**Comment by ShrutikaK2911** [comment]
can you please assign this issue to me

**Comment by ShrutikaK2911** [comment]
can I work on graph data structure

**Comment by jhoffman222** [comment]
yes you may

**Comment by Avnish010** [comment]
Can you  please assign this issue to me .I will add some advanced data structures like  Binary indexed Tree ,Trie and Disjoint Set Union.

**Comment by restrictedCodex** [comment]
hey @jhoffman222 I will add Graph implementations in Golang can you assign me this issue
